root/src/main/java/com/example/servingwebcontent/ConnectDatabase.java

```diff
@@ -61,11 +61,106 @@ public void movieInsert (Movie newMovie) {
 			preparedStmt.setString (i++, newMovie.getFilmRating());
 			preparedStmt.execute();
 		} catch (SQLException e) { 
-			throw new RuntimeException("failed to insert into db", e);
+			throw new RuntimeException("failed to insert into movie table", e);
 		}
 
 	}
 
+	public ResultSet getRating (String id) {
+		ResultSet rs = null;
+		String query = "select * from ratings where id=?";
+		try {
+			PreparedStatement stmt = con.prepareStatement(query);
+			stmt.setString(1, id);
+			rs = stmt.executeQuery();
+		} catch (SQLException e) { 
+			throw new RuntimeException("failed to query ratings table", e); 
+		}
+
+		return rs;
```

Comment:
I just realized that these get methods are all returning result sets. We should translate them to their corresponding business objects so they're easy to use